### PR TITLE
Re-wrote the guide on how to run MS4W

### PR DESCRIPTION
**Pull request by SamDesmondKing**
Updated the MS4W guide to mirror the MS4U section.

Added detailed instructions for how to run MS4W in a scheduled task.

### CODE REVIEW

**Review by jmariomejiap — APPROVED** [review]
This is looking really great @SamDesmondKing 
.  I added a few questions/suggestion but it is ready to 🚀

**Review by pranavmalaviya — APPROVED** [review]
Looks good @SamDesmondKing 🚀

File: pages/tidal tools/machine_stats.md [hunk 1]
```diff
@@ -24,67 +24,118 @@ plan data replication time, or other migration metrics on an app-by-app basis.
...
+  ```
+  Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass
+  ```
+For more information see Microsoft documentation [here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-7.2).
+- Machine Stats uses the same credentials to connect to all your subject machines. Each machine should be accessible with the same username and password.
 
-### Running the Script
+### Creating the Password and Servers File
+
+To start, navigate to the `machine_stats\windows` directory and run the script `.\save_password.ps1`. Enter your password when prompted. This will save your credentials securely in a text file.
```
[the remark this answers is not in the document]
Comment (SamDesmondKing) [reply pages/tidal tools/machine_stats.md:67]: It is indeed stored securely, [this](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/convertto-securestring?view=powershell-7.2) is the cmdlet that it's using if you're curious.

File: pages/tidal tools/machine_stats.md [hunk 2]
```diff
@@ -24,67 +24,118 @@ plan data replication time, or other migration metrics on an app-by-app basis.
...
+__Creating the Task__
 
-And there you have it! Your servers will be synced to Tidal Migrations.
+1. Download the scheduled task XML file [here](https://ms4w-scheduled-task-xml.s3.ca-central-1.amazonaws.com/MS4W-scheduled-task.xml). With a few tweaks, we can use this to quickly create our own scheduled task.
+2. Open `Task Scheduler` and select `import task`. Follow the prompts to import the XML file we just downloaded.
+3. Select `Change User or Groups`. Under `Enter the object name to select`, enter the name of the user you’d like to run the script as (for example, `Administrator`).
+4. Navigate to the `Triggers` tab and edit the trigger. Change the start datetime to whenever you’d like the task to start.
+5. Navigate to the `Actions` tab and, if necessary, edit the script name (default is `run_default`).
+6. Hit OK, you’ll be prompted to enter the password for the user you selected in step 3.
+7. That’s it, your script should be executed at the time you chose in step 4, and will repeat every 5 minutes for 24 hours. You can edit this in the ‘Triggers’ section if you'd like to customize it further.
```
[the remark this answers is not in the document]
Comment (SamDesmondKing) [reply pages/tidal tools/machine_stats.md:138]: This is the configuration of the scheduled task defined in the xml file we're providing the user. I figure it's a good default option, but the user will have to change it themselves if they have different needs.

File: pages/tidal tools/machine_stats.md [hunk 3]
```diff
@@ -24,67 +24,118 @@ plan data replication time, or other migration metrics on an app-by-app basis.
...
+
+#### Run Machine Stats in a Scheduled Task
+
+To run Machine Stats in a Windows scheduled task, we must first create the script which the scheduled task will execute. If you want to save the results as json files to the current directory, with a timestamp in the filename, you can use something like this (remember to replace `<path>` with the correct path):
+
+`run_default.ps1`:
+``` 
+$timestamp = Get-Date -Format "HH-mm-ss"
+
+<path>\runner.ps1 `
```
[the remark this answers is not in the document]
Comment (SamDesmondKing) [reply pages/tidal tools/machine_stats.md:105]: Exactly, I really like the idea but there is a bit of work to do.

**Pull request merged**